docs/sections/user_guide/cli/mode_config.rst

```diff
@@ -173,260 +174,233 @@ The examples that follow use namelist files ``values.nml`` and ``values2.nml`` w
           Print all logging messages
 
 Examples
-~~~~~~~~
+^^^^^^^^
 
-The examples that follow use YAML file ``values.yaml`` with content
+The examples that follow use YAML file ``config.yaml`` with content
```

Comment:
I sorted the lines in `config.yaml`, `values1.yaml`, and `values2.yaml` for consistency with one another.